game-core/src/main/java/games/strategy/triplea/ui/screen/Tile.java

```diff
@@ -15,80 +15,66 @@
 
-  private final Image image;
+  private Image image = Util.newImage(TILE_SIZE, TILE_SIZE, true);
   private final Rectangle bounds;
   private final Object mutex = new Object();
   private final Queue<IDrawable> contents = new PriorityQueue<>();
 
   Tile(final Rectangle bounds) {
     this.bounds = bounds;
-    image = Util.newImage((int) bounds.getWidth(), (int) bounds.getHeight(), true);
   }
 
   public boolean needsRedraw() {
-    return isDirty && !isDrawing;
-  }
-
-  /**
-   * Because our Tile-rendering system doesn't use something like "double buffers" on a higher level
-   * (the swing components support it internally, but we don't draw in batch) we expose this
-   * internal mutex to prevent tiles being drawn mid-refresh.
-   */
-  public Object getMutex() {
-    return mutex;
+    return isDirty && !isDrawing.get();
   }
 
   /** Returns the image representing this tile, re-rendering it first if the tile is dirty. */
-  public Image getImage(final GameData data, final MapData mapData) {
-    if (isDirty) {
-      synchronized (mutex) {
-        isDrawing = true;
-        final Graphics2D g = (Graphics2D) image.getGraphics();
-        g.setRenderingHint(RenderingHints.KEY_RENDERING, RenderingHints.VALUE_RENDER_QUALITY);
-        g.setRenderingHint(
-            RenderingHints.KEY_ALPHA_INTERPOLATION,
-            RenderingHints.VALUE_ALPHA_INTERPOLATION_QUALITY);
-        g.setRenderingHint(
-            RenderingHints.KEY_INTERPOLATION, RenderingHints.VALUE_INTERPOLATION_BICUBIC);
-        g.setRenderingHint(
-            RenderingHints.KEY_TEXT_ANTIALIASING, RenderingHints.VALUE_TEXT_ANTIALIAS_ON);
-        draw(g, data, mapData);
-        g.dispose();
-        isDrawing = false;
-      }
+  public void drawImage(final GameData data, final MapData mapData) {
+    if (isDirty && !isDrawing.getAndSet(true)) {
+      final Image backImage = Util.newImage(TILE_SIZE, TILE_SIZE, true);
+      final Graphics2D g = (Graphics2D) backImage.getGraphics();
+      g.setRenderingHint(RenderingHints.KEY_RENDERING, RenderingHints.VALUE_RENDER_QUALITY);
+      g.setRenderingHint(
+          RenderingHints.KEY_ALPHA_INTERPOLATION, RenderingHints.VALUE_ALPHA_INTERPOLATION_QUALITY);
+      g.setRenderingHint(
+          RenderingHints.KEY_INTERPOLATION, RenderingHints.VALUE_INTERPOLATION_BICUBIC);
+      g.setRenderingHint(
+          RenderingHints.KEY_TEXT_ANTIALIASING, RenderingHints.VALUE_TEXT_ANTIALIAS_ON);
+      draw(g, data, mapData);
+      g.dispose();
+      image = backImage;
+      isDrawing.set(false);
     }
-    return image;
   }
 
-  /**
-   * This image may not reflect our current drawables. Use getImage() to get a correct image
-   *
-   * @return the image we currently have.
-   */
-  public Image getRawImage() {
-    synchronized (mutex) {
-      return image;
-    }
+  /** This image may not reflect our current drawables. */
+  public Image getImage() {
+    return image;
```

Comment:
Thoughts to replace this with `@Getter` and move and consolidate the comment to the `image` variable?

Reply:
I'm not even sure if this method needs a comment tbh

Reply:
If the comment is dropped or not, still holds that we could avoid the boiler plate getter as well. Any thoughts?

Reply:
I see now that was done: https://github.com/triplea-game/triplea/pull/6054/commits/147182a50908b93429f4abee6b0c635b58a4f86f